server/presenters/webhook.ts

```diff
@@ -0,0 +1,34 @@
+import { WebhookDelivery, WebhookSubscription } from "@server/models";
+import { Event } from "@server/types";
+
+interface WebhookProps {
+  event: Event;
+  delivery: WebhookDelivery;
+  subscription: WebhookSubscription;
+  payload: { model: unknown; id?: string };
```

Comment:
This is the presented models at this point, so I don't `BaseModel` will work here

I could work it so that we pass the model and the presenter down farther. The one place that is a bit weird is with the `user.invite` event, cause we don't have an actual User model to hydrate or present. That's one reason I did the presenting a bit higher up, in the WebhookProcessor, to accommodate not always having a real domain object